hack/release.toml

```diff
@@ -109,6 +109,23 @@ machine:
   kubelet:
     disableManifestsDirectory: no
 ```
+"""
+
+    [notes.secretbox]
+        title = "Encryption with secretbox"
+        description = """\
+By default new clusters will use secretbox for encryption instead of AESCBC.
+If both are configured secretbox will take precedence.
+Old clusters may keep using AESCBC.
+To enable secretbox you may add an encryption secret at `cluster.secretboxEncryptionSecret`.
+You should keep `aescbcEncryptionSecret` however, even if secretbox is enabled older data will still be encrypted with AESCBC.
+
+Eg:
+
+```yaml
+cluster:
+  secretboxEncryptionSecret: "z01mye6j16bspJYtTB/5SFX8j7Ph4JXxM2Xuu4vsBPM="
```

Comment:
let's drop the example or make it invalid, probably better to offer a way to generate the token, like `dd if=/dev/random of=/dev/stdout bs=32 count=1 | base64`

as otherwise people will copy paste straight into their machine configs :(